src/algebra/field_power.lean

```diff
@@ -214,6 +214,15 @@ calc x ^ (n + 1) = x ^ n * x : zpow_add_one₀ h₀.ne' _
   x ^ m ≤ x ^ n ↔ m ≤ n :=
 (zpow_strict_mono hx).le_iff_le
 
+lemma min_le_of_zpow_le_max {x : K} (hx : 1 < x) {a b c : ℤ}
+  (h_max : x ^ (-c) ≤ max (x ^ (-a)) (x ^ (-b)) ) : min a b ≤ c :=
```

Comment:
Do we already have:
```suggestion
  (h_max : x ^ c ≤ max (x ^ a) (x ^ b) ) : c ≤ max a b :=
```
Which feels like a simpler statement

Reply:
I didn't see anything like that. We do have `pow_le_max_of_min_le`, which is the reverse implication to my lemma.
This "more complicated" statement is what one usually needs when working with discrete valuations (I used it in #12939).